libraries/avatars/src/AvatarData.cpp

```diff
@@ -2106,6 +2106,18 @@ const QUrl& AvatarData::getSkeletonModelURL() const {
     }
 }
 
+QString AvatarData::getSkeletonModelURLFromScript() const {
+    if (isMyAvatar()) {
+        if (!isMyAvatarURLProtected()) {
+            return _skeletonModelURL.toString();
+        } else {
+            return QString();
+        }
+    } else {
+        return QString();
+    }
```

Comment:
could simplify this:

```suggestion
    if (isMyAvatar() && !isMyAvatarURLProtected()) {
        return _skeletonModelURL.toString();
    }

    return QString();
```